scripts/docker-build.sh

```diff
@@ -8,6 +8,8 @@ fi
 DOCKER_REPO=103425057857.dkr.ecr.us-west-2.amazonaws.com/jitsi/spot-client
 
 cd spot-client
+rm -rf node_modules
```

Comment:
`git clean -dfx` ?

Reply:
is that a hard reset that can clean my local files? I'd rather not go that far, as we're doing those builds from local dev repo right now and I have custom scripts sitting there

Reply:
👍